### ISSUE TITLE
Why did modifying release please manifest just create a breaking change

### ISSUE THREAD

**Issue by VividLemon** [google-github-actions/release-please-action#822]
### TL;DR

https://github.com/bootstrap-vue-next/bootstrap-vue-next/actions/runs/6067806713/job/16459946256
https://github.com/bootstrap-vue-next/bootstrap-vue-next/pull/1414

Changing the manifest created a breaking change release for a unnecessary package. Which was rather annoying. If the package was not pre alpha this would be a big issue as I've had issues with "release-as" in the manifest. It's never seemed to work for me.

### Expected behavior

Not to create a breaking change version. 

### Observed behavior

At most the commit message should create a minor version. 

### Action YAML

```yaml
https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/.github/workflows/release-main.yaml
```

### Log output

```text
Posted above
```

**Comment by VividLemon** [comment]
The commits that it grabbed are very old and have already been indexed. So, it's strange that they were grabbed again.

**Comment by chingor13** [comment]
I think this is a race condition issue when tagging a new release and then immediately trying to open a new PR. Example: https://github.com/google-github-actions/release-please-action/pull/911

**Comment by VividLemon** [comment]
Strange. I recall all that I did was commit the manifest. Nothing particularly out of the ordinary
